### PR TITLE
Sync Hamming

### PR DESCRIPTION
**Pull request by fejan-malek**
https://github.com/exercism/php/issues/790

Added missing 6 test cases

### CODE REVIEW

File: exercises/practice/hamming/HammingTest.php [hunk 1]
```diff
@@ -31,45 +9,92 @@ public static function setUpBeforeClass(): void
...
 
-    public function testCompleteHammingDistanceForSmallStrand(): void
+    /**
+     * uuid: 9aed5f34-5693-4344-9b31-40c692fb5592
+     * @testdox Long identical strands
+     */
+    public function testLongIdenticalStrands(): void
     {
-        $this->assertEquals(2, distance('AG', 'CT'));
+        $this->assertEquals(0, distance('GGACTGAAATCTG', 'GGACTGAAATCTG'));
```
Comment (mk-mxp) [inline exercises/practice/hamming/HammingTest.php:45]: For better readability in online editor:

```suggestion
        $this->assertEquals(0, distance(
            'GGACTGAAATCTG',
            'GGACTGAAATCTG'
        ));
```

File: exercises/practice/hamming/HammingTest.php [hunk 2]
```diff
@@ -31,45 +9,92 @@ public static function setUpBeforeClass(): void
...
 
-    public function testSmallHammingDistance(): void
+    /**
+     * uuid: cd2273a5-c576-46c8-a52b-dee251c3e6e5
+     * @testdox Long different strands
+     */
+    public function testLongDifferentStrand(): void
     {
-        $this->assertEquals(1, distance('AT', 'CT'));
+        $this->assertEquals(9, distance('GGACGGATTCTG', 'AGGACGGATTCT'));
```
Comment (mk-mxp) [inline exercises/practice/hamming/HammingTest.php:54]:
```suggestion
        $this->assertEquals(9, distance(
            'GGACGGATTCTG',
            'AGGACGGATTCT'
        ));
```

File: exercises/practice/hamming/.meta/example.php [hunk 3]
```diff
@@ -1,27 +1,5 @@
...
- *
- * For more info review the Concept on strict type checking in the PHP track
- * <link>.
- *
- * To disable strict typing, comment out the directive below.
- */
-
 declare(strict_types=1);
 
 /**
```
Comment (mk-mxp) [inline exercises/practice/hamming/.meta/example.php:5]: I can't mark the lines...

Please adjust the example function to use the students stub interface:

```php
declare(strict_types=1);

function distance(string $strandA, string $strandB): int
{
    if (strlen($strandA) !== strlen($strandB)) {
        throw new InvalidArgumentException('DNA strands must be of equal length.');
    }

    return count(
        array_diff_assoc(
            str_split($strandA),
            str_split($strandB)
        )
    );
}
```

**Review by mk-mxp — APPROVED** [review]
Thanks for helping with that! These are optional changes only.

File: exercises/practice/hamming/.meta/example.php (hunk 3 above)
Comment (fejan-malek) [reply exercises/practice/hamming/.meta/example.php:5]: I also added code doc

**Review by mk-mxp — APPROVED** [review]
Great! Thanks a lot. Looking forward to the next one...

**Pull request merged**